neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
```

Comment:
taky bych to chtěl dát k dekodéru. Nebude to pak ale moc hnusnej soubor?

Btw. co je ctc?